makefile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I do like this make file, but for consistancy should we use bazel? I don't think it needs to be settled prior to this PR merging, but worth discussing.

Reply:
I agree, but I think bazel commands are a kind of (boring) long, and a majority of programmers are familiar with makefile. Hence perhpas we can use make commands,  like `unit-test`, as alias for bazel commands. 

I mean, makefile acts like an interface, and bazel is its backend implementation. Perhaps it can help new contributor get used to, and we can also replace bazel in the future if we find another building candidate. Is it too complicated?

Reply:
Create some alias for the most commonly used commands: code-format, sanity-check, unit-test. They seems enough for new contributor.

Reply:
I think it's fine for us to use make for simple dev tasks. We can always revisit a new build soln if we think its needed in the future